vms/shift/forms.py

```diff
@@ -20,6 +20,17 @@ class Meta:
             'state', 'city', 'address', 'venue'
         ]
 
+    # Shifts are bound to end on the same date so

```

Comment:
would this work if my shift spans two dates?
start before midnight and end after?

Reply:
For that we need to create 2 separate shifts.

Reply:
I think what @pavankm means that say my shift starts at 10PM on day 1 and goes on till 6 AM the next day. Will it work in that situation?

Reply:
If `start_time` and `end_time` are datetime objects or UNIX epochs, this approach will work as it is. If not, it needs to be verified.

Reply:
Yeah I confirmed this in vms meeting, the shift has to end on the same day.
So the case you mentioned will have 2 shifts, 1 from 10pm to 11:59pm and 2nd from 00:00 to 06:00.
Just to confirm again I'll tag vms people here.
@ayushgarg1804 @abhishekarya286 @kamsuri @anjali-dhanuka

Reply:
Alright, looks good then!